File: Python/Logic - 2/lone_sum.py
def lone_sum(a, b, c):
    
    if a==b==c:
        
        return 0
    
    elif b==c:
        
        return a
    
    elif a==c:
        
        return b
    
    elif a==b:
        
        return c
    
    else:
        
        return a+b+c

File: Java/Logic - 2/loneSum.java
public int loneSum(int a, int b, int c) {

    if(a==b && b==c) {

        return 0;

    }

    if(a==b) {

        return c;

    }

    else if(b==c) {

        return a;

    }

    else if(c==a) {

        return b;

    }

    else {

        return a+b+c;

    }

}